x-pack/plugins/apm/public/components/shared/time_comparison/use_time_range_comparison.ts

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import moment from 'moment';
+import { EuiTheme } from 'src/plugins/kibana_react/common';
+import { getDateDifference } from '../../../../common/utils/formatters';
+import { useTheme } from '../../../hooks/use_theme';
+
+function getChartTheme(theme: EuiTheme) {
```

Comment:
this is specifically for comparison, right?
```suggestion
function getComparisonChartTheme(theme: EuiTheme) {
```